taipy/gui/gui.py

```diff
@@ -445,6 +445,10 @@ def __process_content_provider(self, state: State, path: str, query: t.Dict[str,
                     if isinstance(content, PlotlyFigure):
 
                         def get_plotly_content(figure: PlotlyFigure):
+                            mapbox_token = get_mapbox_token()
```

Comment:
I think we wanted to support map token *not only* for figure but also for plotly.js based charts